include/tables/object_table.h

```diff
@@ -0,0 +1,652 @@
+/**
+ * Object Table
+ *
+ * DEFINE_OBJECT should be used for normal objects
+ *    - Argument 0: Name of the object segment in spec
+ *    - Argument 1: Enum value for this object
+ *
+ * DEFINE_OBJECT_UNSET and DEFINE_OBJECT_NULL are needed to define empty entries from the original game
```

Comment:
IMO this needs to be more specific, especially since one is only used once and it doesn't really do what it sounds like from the name.

Reply:
Added a few extra explanations, is it better?

Reply:
Yeah, I think so. I'm a bit torn on NULL: I see that it's for an object that's not in the rom but the name sounds like it would be a null pointer rather than having the beginning and end addresses the same.

Reply:
It is the OoT name. Do you have any better suggestion? maybe `_NO_SIZE`?

Reply:
Or `SIZE_ZERO` possibly.